Azure Services/Azure Monitor/Agents/Migration Tools/DCR Config Generator/WorkspaceConfigToDCRMigrationTool.ps1

```diff
@@ -609,18 +739,54 @@ function Get-DCRStream
         "WindowsPerformanceCounter" { $stream += "Microsoft-Perf"; Break }
         "LinuxPerformanceObject" { $stream += "Microsoft-Perf"; Break }
         "WindowsEvent" { $stream += "Microsoft-Event"; Break }
-        "LinuxSyslog" { $stream += "Microsoft-Syslog" }
+        "LinuxSyslog" { $stream += "Microsoft-Syslog"; Break }
+        "CustomLog" 
+        {
+            $workspaceDataSourceList = Get-AzOperationalInsightsDataSource -ResourceGroupName $ResourceGroupName -WorkspaceName $WorkspaceName -Kind $dataSourceType
+            foreach($dataSource in $workspaceDataSourceList)
+            {
+                if ($dataSource.Properties -ne $null -and $dataSource.Properties.customLogName -ne $null)
+                {
+                    $stream += "Custom-" + $dataSource.Properties.customLogName
+                }
+            }
+            Break
+        }
+
+        "IISLog" { $stream += "Microsoft-W3CIISLog" }
```

Comment:
Why not add a Break here too?

Reply:
Does it matter? Since it's the last switch case not adding a break should not matter

Reply:
It's a question of style and consistency. I'll ask the reverse question, does it matter if we add it?

Reply:
It doesn't matter. I added it thank you